src/content/platform-integration/ios/setup.md

```diff
@@ -0,0 +1,253 @@
+---
+title: Set up iOS development
+short-title: Set up iOS
+description: >-
+  Configure your development environment to
+  run, build, and deploy Flutter apps for iOS devices.
+---
+
+Learn how to set up your development environment
+to run, build, and deploy Flutter apps for iOS devices.
+
+:::note
+If you haven't set up Flutter already,
+visit and follow the [Get started with Flutter][] guide first.
+
+If you have already installed Flutter,
+ensure that it's [up to date][].
+:::
+
+[Get started with Flutter]: /get-started/install
+[up to date]: /install/upgrade
+
+## Install Xcode
+
+With Xcode, you can run Flutter apps on
+an iOS physical device or on the iOS Simulator.
+
+If you haven't done so already,
+install and set up [Xcode][].
+
+[Xcode]: https://developer.apple.com/xcode/
+
+Configure Xcode's command-line tools:
+
+```console
+$ sudo sh -c 'xcode-select -s /Applications/Xcode.app/Contents/Developer && xcodebuild -runFirstLaunch'
+```
+
+Sign the Xcode license agreement:
+
+```console
+$ sudo xcodebuild -license
+```
+
+## Install Rosetta
+
+If you are developing on an [Apple Silicon][] (ARM) Mac,
+[install Rosetta 2][].
+
+```console
+sudo softwareupdate --install-rosetta --agree-to-license
+```
+
+[Apple Silicon]: https://support.apple.com/en-us/116943
+[install Rosetta 2]: https://support.apple.com/en-us/102527
+
+## Install CocoaPods
+
+To support [Flutter plugins][] that use native iOS code,
+install [CocoaPods][cocoapods].
+CocoaPods bundles various dependencies that
+support connecting Flutter to native iOS code.
+
+Install `cocoapods` following the
+[CocoaPods installation guide][cocoapods].
+
+[cocoapods]: https://guides.cocoapods.org/using/getting-started.html#installation
+[Flutter plugins]: /packages-and-plugins/developing-packages#types
+
+## Configure your target iOS device
+
+We recommend starting with the iOS Simulator as
+it's easier to get setup than a physical iOS device.
+However, you should also test your app on an actual
+physical device.
+
+{% tabs "ios-simulator-or-physical-device" %}
+{% tab "Simulator" %}
+
+### Configure your iOS Simulator {:.no_toc}
+
+Use the following steps:
+
+1. To install the iOS Simulator, run the following command:
+
+    ```console
+    $ xcodebuild -downloadPlatform iOS
```

Comment:
This `xcodebuild -downloadPlatform iOS` instruction is needed for all iOS development, it should be moved out of any simulator section.